### PR TITLE
Improved consistency

### PR DESCRIPTION
**Pull request by ErikSchierboom**

### CODE REVIEW

File: docs/getting-involved-in-a-track.md [hunk 1]
```diff
@@ -80,14 +80,14 @@ README in the track repository.
 As you solve exercises on the site, pay attention to what you like and dislike
 about the exercises.
 
-### Improving Exercise READMEs
+### Improving Exercise README's
```
Comment (kytrinyx) [inline docs/getting-involved-in-a-track.md]: I'm always unsure of this. The apostrophe is for possessive, right?

**Comment by kytrinyx** [comment]
Niiice! Thank you. I've only got the one question about READMEs vs README's.

**Comment by ErikSchierboom** [comment]
You were right about the READMEs (see https://en.wiktionary.org/wiki/readmes#English). I've also found some other broken links and fixed them.

File: docs/getting-involved-in-a-track.md [hunk 2]
```diff
@@ -100,23 +100,23 @@ directory in the language track repository for the directory named after the
 slug of the exercise. The test file will be right there.
 
 That said, some test suites are generated based on the `canonical-data.json`
-file found in [x-common][]. The README for the language track repository
+file found in [x-common][https://github.com/exercism/x-common]. The README for the language track repository
```
Comment (kytrinyx) [inline docs/getting-involved-in-a-track.md]: This fix breaks the link.

The `[x-common][]` is a shorthand for `[x-common][x-common]`, which references the `[x-common]: http://...` definition, which is defined at the top of the file.

File: docs/getting-involved-in-a-track.md [hunk 3]
```diff
@@ -100,23 +100,23 @@ directory in the language track repository for the directory named after the
...
-- hash out all the details together
-- figure out the necessary changes to the `canonical-data.json` for the
-  exercise (if it exists)
-- draft an issue that can be submitted to all the tracks that implement the
-  exercise using the [blazon][] tool (which automates the tedious parts).
+- Hash out all the details together.
+- Figure out the necessary changes to the `canonical-data.json` for the
+  exercise (if it exists).
+- Draft an issue that can be submitted to all the tracks that implement the
+  exercise using the [blazon][https://github.com/exercism/blazon] tool (which automates the tedious parts).
```
Comment (kytrinyx) [inline docs/getting-involved-in-a-track.md]: Ditto—this was using the reference defined at the top of the file.

File: docs/getting-involved-in-a-track.md [hunk 4]
```diff
@@ -100,23 +100,23 @@ directory in the language track repository for the directory named after the
...
 That said, some test suites are generated based on the `canonical-data.json`
-file found in [x-common][]. The README for the language track repository
+file found in [x-common][https://github.com/exercism/x-common]. The README for the language track repository
 should tell you what you need to know.
 
 For every change that you make to the test suite, ask yourself: **Should this
 change also affect other language tracks that implement the same exercise?**
 
-If you think it might, open a discussion in the [x-common][] repository and get
+If you think it might, open a discussion in the [x-common][https://github.com/exercism/x-common] repository and get
```
Comment (kytrinyx) [inline docs/getting-involved-in-a-track.md]: This is broken, too.

File: docs/getting-involved-in-a-track.md [hunk 5]
```diff
@@ -100,23 +100,23 @@ directory in the language track repository for the directory named after the
...
 other track implementors' feedback on the subject.
 
 If this change should affect the broader Exercism curriculum, then use the
-issue in x-common to
+issue in [x-common](https://github.com/exercism/x-common) to:
```
Comment (kytrinyx) [inline docs/getting-involved-in-a-track.md]: This can use the reference at the top of the file (`[x-common][]`)

**Comment by ErikSchierboom** [comment]
@kytrinyx My bad. The PR has been updated.

**Comment by kytrinyx** [comment]
Awesome, thanks!

**Pull request merged**